Sources/Rendering/OpenGL/VolumeMapper/test/testLighting.js

```diff
@@ -91,16 +90,18 @@ test.onlyIfWebGL('Test Lighted Volume Rendering', (t) => {
       renderer.getActiveCamera().zoom(1.4);
       renderer.resetCameraClippingRange();
       renderer.updateLightsGeometryToFollowCamera();
+      mapper.setOnLightingActivated(() => {
```

Comment:
this will become `mapper.onLightingActivated(() => {`

Reply:
Done.